test/functionalTest/cases/1259_new_json_for_v2/put_attribute_value.test

```diff
@@ -0,0 +1,582 @@
+payload='true'
+orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --in text/plain --out application/json
+echo
+echo
+
+
+echo '10. Check that A1 == true'
+echo '========================='
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo '11. PUT /v2/entities/E1/attrs/A1/value with TEXT payload TRUE'
+echo '============================================================='
+payload='TRUE'
+orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --in text/plain --out application/json
+echo
+echo
+
+
+echo '12. Check that A1 == true'
+echo '========================='
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo '13. PUT /v2/entities/E1/attrs/A1/value with TEXT payload True'
+echo '============================================================='
+payload='True'
+orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --in text/plain --out application/json
+echo
+echo
+
+
+echo '14. Check that A1 == true'
+echo '========================='
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo '15. PUT /v2/entities/E1/attrs/A1/value with TEXT payload false'
+echo '=============================================================='
+payload='false'
+orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --in text/plain --out application/json
+echo
+echo
+
+
+echo '16. Check that A1 == false'
+echo '=========================='
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo '17. PUT /v2/entities/E1/attrs/A1/value with TEXT payload FALSE'
+echo '=============================================================='
+payload='FALSE'
+orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --in text/plain --out application/json
+echo
+echo
+
+
+echo '18. Check that A1 == false'
+echo '=========================='
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo '19. PUT /v2/entities/E1/attrs/A1/value with TEXT payload False'
+echo '=============================================================='
+payload='False'
+orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --in text/plain --out application/json
+echo
+echo
+
+
+echo '20. Check that A1 == false'
+echo '=========================='
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo '21. PUT /v2/entities/E1/attrs/A1/value with TEXT payload 123'
+echo '============================================================'
+payload='123'
+orionCurl --url /v2/entities/E1/attrs/A1/value --payload "$payload" -X PUT --in text/plain --out application/json
+echo
+echo
+
+
+echo '22. Check that A1 == 123'
+echo '========================'
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo '23. PUT /v2/entities/E1/attrs/A1/value with TEXT payload 3.14'
+echo '============================================================='
+payload='3.14'
```

Comment:
Maybe including a text with negative number (pe. -3.14) and a test with cientific notation (pe. 3.2e2) ?